sprint-planning/estimation/readme.md

```diff
@@ -1,8 +1,87 @@
 # Estimation
 
-## Content TODO's
+When a team adopts agile, the process of estimating new work becomes more effective.

```

Comment:
The team needs to be **durable** for the estimation exercise to work effectively and predict velocity.

Reply:
Good point. I need to call that out because each time a team reorganizes, the thinking resets and the baseline moves.